oyatplum/react-todo/src/component/todoitem.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const TodoItem = (props) => {
```

Comment:
```suggestion
const TodoItem = ({ itemId, item, todoList, setTodoList }) => {
```
[구조분해할당](https://chanhuiseok.github.io/posts/js-10/)으로 props를 더 보기 좋게 가져올 수 있어요.